config.json

```diff
@@ -127,9 +127,7 @@
         "binaries":{
           "description": "Only builds binaries. It doesn't restore packages.",
           "settings":{
-            "RestoreDuringBuild": false,
-            "BuildPackages": false,
-            "BuildTests": false
+            "RestoreDuringBuild": false
```

Comment:
I'm making the correspondent changes to build tools in order to use the run tool. This change doesn't change the behavior of build, but cleans the config.json file to use the build options that we have in build tools.
I could create a different PR for it, but I'm not sure if it is worth it.